ci/stable_driver.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash --login
+
+my_dir="$( cd "$( dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
+
+# ==============================================================================
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 -t <target> -h"
+  echo
+  echo "  -t  target/machine script is running on    DEFAULT: $(hostname)"
+  echo "  -h  display this message and quit"
+  echo
+  exit 1
+}
+
+# ==============================================================================
+# First, set up runtime environment
+
+export TARGET="$(hostname)"
+
+while getopts "t:h" opt; do
+  case $opt in
+    t)
+      TARGET=$OPTARG
+      ;;
+    h|\?|:)
+      usage
+      ;;
+  esac
+done
+
+case ${TARGET} in
+  hera | orion)
+    echo "Running stability check on $TARGET"
+    source $MODULESHOME/init/sh
+    source $my_dir/${TARGET}.sh
+    module purge
+    module use $GDAS_MODULE_USE
+    module load GDAS/$TARGET
+    module list
+    ;;
+  *)
+    echo "Unsupported platform. Exiting with error."
+    exit 1
+    ;;
+esac
+
+# ==============================================================================
+# clone a fresh copy of the stable branch
+datestr="$(date +%Y%m%d)"
+repo_url="https://github.com/NOAA-EMC/GDASApp.git"
+stableroot=$GDAS_CI_ROOT/stable
+
+mkdir -p $stableroot/$datestr
+cd $stableroot/$datestr
+git clone $repo_url
+cd GDASApp
+git checkout feature/stable-nightly
+
+# ==============================================================================
+# merge in develop
+git merge develop
+
+# ==============================================================================
+# update the hashes to the most recent
+$my_dir/stable_mark.sh $stableroot/$datestr/GDASApp
+
+# ==============================================================================
+# run the automated testing
+$my_dir/run_ci.sh -d $stableroot/$datestr/GDASApp -o $stableroot/$datestr/output
+ci_status=$?
+if [ $ci_status -eq 0 ]; then
+  # push a new commit to the stable branch
+  cd $stableroot/$datestr/GDASApp
+  git add CMakeLists.txt
+  git commit -m "Update to new stable build on $datestr"
+  git push
+  echo "Stable branch updated"
+else
+  # do nothing
+  echo "Testing failed, stable branch will not be updated"
+fi
+# ==============================================================================
+# publish some information to RZDM for quick viewing
+# THIS IS A TODO FOR NOW
```

Comment:
I put this as a todo but before I spend any time on it, @guillaumevernieres @RussTreadon-NOAA do you see any use in a 'dashboard' that gets updated? I was thinking having a page that has all of the commit hashes and the last time the stable branch was updated on a simple HTML page (any other suggested items?) that is rsync'd to RZDM. But if you think that is unnecessary then I won't worry about it. As I type that out, I guess all of this could be found from GitHub so maybe it's not useful after all... thoughts?